core/data-transformers/page-info-transformer.ts

```diff
@@ -0,0 +1,22 @@
+import { ResultOf } from 'gql.tada';
+
+import { CursorPaginationInfo } from '@/vibes/soul/primitives/cursor-pagination';
+import { PaginationFragment } from '~/client/fragments/pagination';
+
+export const defaultPageInfo = {
+  hasNextPage: false,
+  hasPreviousPage: false,
+  startCursor: null,
+  endCursor: null,
+};
+
+export function pageInfoTransformer(
+  pageInfo: ResultOf<typeof PaginationFragment>,
+): CursorPaginationInfo {
+  return {
+    startCursorParamName: 'before',
+    startCursor: pageInfo.startCursor,
+    endCursorParamName: 'after',
+    endCursor: pageInfo.endCursor,
```

Comment:
Could there be a situation where we have two paginations in the same page, and since we can't modify the param names using this transformer, we'd be screwed 🤔 .

I guess for now lets not think about this because it is not the case.